.github/workflows/cr.yml

```diff
@@ -0,0 +1,45 @@
+name: CR
```

Comment:
I don't understand why we need this and can't do that in the comment workflow?

Reply:
I'm newbie to gh actions/workflows, feel free to play with other layouts.

You can check vitepress integration, brc-cc did some changes about the publish comment, actions cannot fire workflows, maybe can be done: 
- original PR https://github.com/vuejs/vitepress/pull/406
- including cr-comment workflow https://github.com/vuejs/vitepress/commit/1d629c5c4fdb582c357d9d917a1ab0d3df738687

Reply:
You can check vite publish-comment workflow, I use it with no luck.

Reply:
Ah, yes! Vite workflow is what I expected